arroyo-rpc/build.rs

```diff
@@ -10,7 +10,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .file_descriptor_set_path(out_dir.join("api_descriptor.bin"))
         .type_attribute(".", "#[derive(serde::Serialize, serde::Deserialize)]")
         .type_attribute(".", "#[serde(rename_all = \"camelCase\")]")
-        .compile(&["proto/api.proto"], &["api"])
+        .compile(&["proto/api.proto"], &["proto/"])
```

Comment:
what is this change doing?

Reply:
I was seeing errors like
       Dirty arroyo-rpc v0.1.0 (/Users/jacksonnewhouse/code/arroyo/arroyo-rpc): the file `/Users/jacksonnewhouse/code/arroyo/arroyo-rpc/api` is missing

Which was causing everything that depends on arroyo-rpc to rebuild on every run. Looking at the code for compile_protos, it should be the directory that the proto definitions are in.